core/controllers/feedback.py

```diff
@@ -24,11 +24,13 @@
 
 transaction_services = models.Registry.import_transaction_services()
 
+
 class ThreadListHandler(base.BaseHandler):
     """Handles operations relating to feedback thread lists."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    @acl_decorators.open_access
```

Comment:
Wouldn't can_view_exp_feedback be better than can_play in that case? In the sense that that's basically what this is doing?

Unless the name is wrong, in which case we might want to change it?